boto3/docs/method.py

```diff
@@ -37,19 +37,25 @@ def document_model_driven_resource_method(
     if resource_action_model.resource:
         if 'return' in section.available_sections:
             section.delete_section('return')
+        resource_type = resource_action_model.resource.type
```

Comment:
Since resource_action_model.resource.type is now referenced 4 times, I created a new variable called resource_type